kraken/node_actions/node_scenarios.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sys
+import time
+import json
+import yaml
+import boto3
```

Comment:
Can we create an abstract node scenarios class and have cloud specific implementing classes for it that use the cloud specific libraries like boto3 (e.g. aws_node_scenarios)?  The config can contain an option for the cloud type to use and the client can just deal with generic scenarios like start/stop, reboot, termination.

Reply:
Yeah, this sounds good. I will make the changes to implement the same. Thank you for the feedback! @mffiedler